__tests__/git.test.ts

```diff
@@ -178,7 +178,7 @@ describe('git', () => {
       }
     })
 
-    it('should fail if the build folder begins with a /', async () => {
+    it('should not fail if the build folder begins with a /', async () => {
```

Comment:
I don't think this test is useful anymore as it's not checking for anything. I believe there's already another test which ensures all of the commands run already in this suite. Probably safe to delete this one.